modules/dind/examples_test.go

```diff
@@ -0,0 +1,51 @@
+package dind_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/docker/docker/client"
+	"github.com/testcontainers/testcontainers-go/modules/dind"
+	"log"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+func ExampleRun() {
+	// runDinDContainer {
+	ctx := context.Background()
+
+	dindContainer, err := dind.Run(ctx, "docker:28.0.1-dind")
+	defer func() {
+		if err := testcontainers.TerminateContainer(dindContainer); err != nil {
+			log.Printf("failed to terminate container: %s", err)
+		}
+	}()
+	if err != nil {
+		log.Printf("failed to start container: %s", err)
+		return
+	}
```

Comment:
suggestion: let's close the code snippet here so we can create more of them
```suggestion
	}
	// }
```